node/src/database/db_migrations.rs

```diff
@@ -320,16 +307,18 @@ mod tests {
 
     #[derive(Default)]
     struct DBMigrationUtilitiesMock {
-        update_schema_version_params: Arc<Mutex<Vec<String>>>,
+        update_schema_version_params: Arc<Mutex<Vec<usize>>>,
         update_schema_version_results: RefCell<Vec<rusqlite::Result<()>>>,
         execute_upon_transaction_params: Arc<Mutex<Vec<Vec<String>>>>,
         execute_upon_transaction_result: RefCell<Vec<rusqlite::Result<()>>>,
```

Comment:
Probably should be `execute_upon_transaction_resultS` to be like the other results fields